### ISSUE TITLE
AST Audioset Training Time and Hardware

### ISSUE THREAD

**Issue by justinluong** [YuanGongND/ast#116]
Hi,

I hope this message finds you well. I am currently working on my thesis and have found your work on AST invaluable.

I am using your pretrained model (Full AudioSet, 10 tstride, 10 fstride, with Weight Averaging) as a baseline for comparison.

Would you be able to provide some details on the hardware used to train this model, as well as the approximate training time? This information would be incredibly helpful for the context of my research.

Thank you very much for your time and assistance.

**Comment by YuanGongND** [comment]
hi there,

We also released the training log of that experiment at https://github.com/YuanGongND/ast/blob/master/egs/audioset/exp/test-full-f10-t10-pTrue-b12-lr1e-5/log_2090852.txt

All information can be found in the log. 

Specifically, the GPU is 4 x NVIDIA TITAN X (12GB each GPU), an old GPU.

Training time per epoch = 118072 seconds, we trained 5 epochs, which is about 7 days. With modern GPUs, it will be faster.

I recommend checking the mAP immediately after the 1st epoch and seeing if it matches our log.

-Yuan
